libbeat/docs/breaking.asciidoc

```diff
@@ -11,12 +11,22 @@ changes, but there are breaking changes between major versions (e.g. 5.x to
 
 See the following topics for a description of breaking changes:
 
+* <<breaking-changes-6.7>>
 * <<breaking-changes-6.3>>
 * <<breaking-changes-6.2>>
 * <<breaking-changes-6.1>>
 * <<breaking-changes-6.0>>
 * {auditbeat-ref}/auditbeat-breaking-changes.html[Breaking changes in Auditbeat 6.2]
 
+=======
```

Comment:
```suggestion
```

Reply:
@ph These must have been left over from when you resolved file conflicts? Need to remove because this line causes build errors.